server/src/main/java/org/elasticsearch/common/breaker/CircuitBreaker.java

```diff
@@ -84,6 +84,16 @@ enum Durability {
      */
     void addEstimateBytesAndMaybeBreak(long bytes, String label) throws CircuitBreakingException;
 
+    /**
+     * Check the real memory circuit breaker. It is a NOOP if
+     * {@link org.elasticsearch.indices.breaker.HierarchyCircuitBreakerService#USE_REAL_MEMORY_USAGE_SETTING} is set to false,
+     * otherwise it trips if the current memory usage cannot be lowered below thw limit.
```

Comment:
Typo: thw -> the
Maybe also report here this method is forbidden/temporary/should not be used?